packages/model-viewer/src/three-components/ARRenderer.ts

```diff
@@ -415,40 +415,15 @@ export class ARRenderer extends EventDispatcher {
 
   private updateView(view: XRView) {
     const scene = this.presentedScene!;
+    const xr = this.threeRenderer.xr as any;
```

Comment:
three.js types are out of date.

Reply:
My current setup, to avoid `any` 😢 :
```ts
type ExtendedRenderer = WebGLRenderer & {
  cameraAutoUpdate: boolean
  xr: {
    updateCamera: (camera: Camera) => void
    getCamera: () => Camera
  }
}
...
const renderer = new WebGLRenderer({
  canvas,
  alpha: true,
  antialias: true,
  powerPreference: 'high-performance' as WebGLPowerPreference,
  preserveDrawingBuffer: true,
}) as ExtendedRenderer
```